internals/daemon/api.go

```diff
@@ -117,3 +117,7 @@ func v1SystemInfo(c *Command, r *http.Request, _ *userState) Response {
 	}
 	return SyncResponse(result)
 }
+
+func Register(c *Command) {
+	api = append(api, c)
+}
```

Comment:
Please have a look at this snippet:
https://github.com/canonical/pebble/blob/00bcd1f7dda570bcdf42600545ddc1707d24365e/internals/daemon/api.go#L27-L97

I had a similar issue while coming up with an API for adding commands to the CLI. Initially, I came up with the `cli.AddCommand()` method, but my proposal was rejected. One of the major points that came up were that it would be easier to just export the `Command`s slice. This way, on applications extending Pebble, we could have a slice of `Command`s at the top side of the `main.go` file, exposing all the commands' information in a single place and acting as a reference that clearly exposes all available commands upfront to the developer.

I'd like to see a similar approach here. In applications extending Pebble, we should be able to just create a `Command` slice that we can append to the Pebble one that exposes all the custom API endpoints like in the snippet above, so that it is immediately obvious to developers what's in the box (: